src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -1170,6 +1168,14 @@
       <Link>Common\System\Threading\Tasks\TaskToApm.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(OptimizeForSize)' != true">
```

Comment:
Nit: We put single quotes around true. Same comment applies to line 1176.
(See line 21 for example.)